llvm/test/CodeGen/PowerPC/llvm.sincos.ll

```diff
@@ -49,3 +49,49 @@ define { ppc_fp128, ppc_fp128 } @test_sincospi_ppcf128(ppc_fp128 %a) {
   %result = call { ppc_fp128, ppc_fp128 } @llvm.sincospi.ppcf128(ppc_fp128 %a)
   ret { ppc_fp128, ppc_fp128 } %result
 }
+
+; FIXME: Recognise this as a tail call and omit the stack frame:
+define void @test_sincos_ppcf128_tail_call(ppc_fp128 %a, ptr noalias %out_sin, ptr noalias %out_cos) {
```

Comment:
This should return the raw structure type

Reply:
That won't result in something that could be a tail call on any target that uses the (semi)-standard GNU `sincos` function (since it'll need to emit loads after the call).